README.md

```diff
@@ -2351,6 +2352,44 @@ Checks if string ends with the given target string.
 
 **[⬆ back to top](#quick-links)**
 
+### _.isString
+:heavy_exclamation_mark:`Not in Underscore.js`
+Checks if string ends with the given target string.
+
+  ```js
+  // Lodash
+  _.isString('abc');
+  // => true
+ 
+  _.isString(123);
+  // => false
+  
+  _.endsWith('abc', 'b', 2);
```

Comment:
Delete this line :)

Reply:
which one to be precise

Reply:
Look like you copied from endsWith and forgot remove

Reply:
oh right